### ISSUE TITLE
Port to ESP8266

### ISSUE THREAD

**Issue by Ivoah** [cnlohr/avrjslinux#1]
I'd be cool to have this ported to the ESP8266, and it'd be almost useful too!

**Comment by cnlohr** [comment]
You are correct, this would be pretty cool.  If I ever get an excuse to do it, I will :)

**Comment by Ivoah** [comment]
Coolio!
